src/app/users/get-user.ts

```diff
@@ -33,8 +32,9 @@ export const getUserForLogin = async ({
         return user
       }
       addAttributesToCurrentSpan({
-        [ENDUSER_ALIAS]: user.username,
+        [SemanticAttributes.ENDUSER_ID]: user.id,
```

Comment:
The `ENDUSER_ID` is already added https://github.com/GaloyMoney/galoy/blob/main/src/servers/graphql-server.ts#L78 `ENDUSER_ALIAS` is not.

Reply:
there is no longer a concept of `ENDUSER_ALIAS`, should be account related now. should I just delete this for now?

Reply:
Sure delete it then